qiskit_experiments/data_processing/nodes.py

```diff
@@ -28,11 +27,18 @@ class AverageData(DataAction):
     """A node to average data representable as numpy arrays."""
 
     def __init__(self, axis: int, validate: bool = True):
-        """Initialize a data averaging node.
+        r"""Initialize a data averaging node.
 
         Args:
             axis: The axis along which to average.
             validate: If set to False the DataAction will not validate its input.
+
+        Notes:
+            If standard error of input array is not populated, this node will compute
+            standard error of the mean, i.e. the standard deviation of the datum divided by
+            :math:`\sqrt{N}` where :math:`N` is the number of data points.
+            Otherwise standard error is computed by quadratic sum of the errors of input data
+            divided by the number of data points, as usual error propagation.
```

Comment:
```suggestion
            If the input array has no standard error, then this node will compute the
            standard error of the mean, i.e. the standard deviation of the datum divided by
            :math:`\sqrt{N}` where :math:`N` is the number of data points.
            Otherwise the standard error is given by the square root of :math:`N^{-1}` times 
            the sum of the squared errors.
```